src/index.js

```diff
@@ -141,70 +143,82 @@ const getReportFilename = (writeReports = false) => {
   }
 }
 
-const menu = async (category, selectAll = false) => {
+const menu = async (categories, selectAll = false) => {
   clearConsole()
-
-  const tests = listTestcases(TestCases[category])
-  const questions = [
-    {
-      type: 'checkbox-plus',
-      name: 'TESTS',
-      message: [
-        [
-          'Select the test(s) you want to execute using the',
-          Chalk.yellow('spacebar'),
-          'then press',
-          Chalk.yellow('enter'),
-          'to start the Testrunner!',
-          'To go',
-          Chalk.red('back'),
-          'press enter without selecting a test.',
-        ].join(' '),
-        [Chalk.yellow('➜'), Chalk.dim('Search by title / description:')].join(' '),
-      ].join('\n'),
-      searchable: true,
-      source: (selected, input) => {
-        return new Promise(function(resolve) {
-          resolve([
-            'Enable writing reports to file',
-            'Run all',
-            new Inquirer.Separator(),
-            ...tests.filter(test => {
-              return test.name.toLowerCase().includes(input.toLowerCase())
-            }),
-          ])
-        })
+  let displayTestCaseList = []
+  let testCaseList = []
+  categories.forEach(category => {
+    let testCaseListFromCategory = TestCases[category] //Get test case list from each category
+    if (testCaseListFromCategory && testCaseListFromCategory.length) {
+      testCaseList.push(...TestCases[category]) //Create test case list for all the selected categories to run
+      displayTestCaseList.push(...listTestcases(TestCases[category])) //Create testcase list for all the selected categories to display
+    }
+  })
+  if (categories.includes('Enable writing reports to file')) {
+    writeReports = true
+  }
+  if (categories.length > 1 || categories.includes('Run all Plugin Tests')) {
```

Comment:
It feels a bit like a hack to check if the selected category is actually the run all plugin tests. Why dont handle this in the 

```
Inquirer.prompt(questions).then(answers => {
```

section?